samples/sensor/grow_r502a/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _grow_r502a:
+
+GROW_R502A Fingerprint Sensor
+#############################
+
+Overview
+********
+
+This sensor sample shows template count inside fingerprint sensor and do the following,
+Finger enrollment ,fingerprint Matching and deleting. Sensor LED will blink accordingly.
+
+enrollment and matching should work only when trigger mode is set.
+
+Wiring
+*******
+
+This sample uses an external breakout for the sensor.  A devicetree
+overlay must be provided to identify the UART bus and the GPIO
+used to control the sensor. Sensor can be powered using mentioned optional GPIO.
+
+References
+**********
+
+For more information about the GROW_R502A Fingerprint Sensor see
+http://en.hzgrow.com/product/143.html
+
+Building and Running
+********************
+
+After providing a devicetree overlay that specifies the sensor location,
+build this sample app using:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/sensor/grow_r502a
+   :board: nrf52840dk_nrf52840
+   :goals: build flash
+
+Sample Output
+=============
+
+.. code-block:: console
+
+    *** Booting Zephyr OS build v2.7.99-2153-g2cbc14c13ee5  ***
+    template count : 2
+    Fingerprint Deleted at ID #3
+    Waiting for valid finger to enroll as ID #2
+    Place your finger
+    Fingerprint successfully stored at #2
+    Matched ID : 2
+    confidence : 217
+    Matched ID : 2
+    confidence : 166
+    Matched ID : 2
+    confidence : 169
+
+<repeats endlessly>
```

Comment:
Please put this in the code block or turn it into a complete sentence instead

Reply:
Thanks for the review, resolved this and all the above mentioned comments.